source/_components/alarm_control_panel.egardia.markdown

```diff
@@ -32,17 +32,18 @@ Configuration variables:
 - **host** (*Required*): The local IP address of the Egardia/Woonveilig alarm panel.
 - **username** (*Required*): Username for the Egardia/Woonveilig account.
 - **password** (*Required*): Password for Egardia/Woonveilig account.
+- **version** (*Optional*): The version of the Egardia system. `GATE-01` and `GATE-02` are currently supported. Defaults to `GATE-01`.
 - **port** (*Optional*): The port of the alarm panel. Defaults to 80.
 - **name** (*Optional*): Name to use for the alarm panel. Defaults to `Egardia`.
 - **report_server_enabled** (*Optional*): Enable reporting by server. Defaults to `False`.
-- **report_server_port** (*Optional*): Port of the Egardia server. Defaults to 85.
+- **report_server_port** (*Optional*): Port of the Egardia server. Defaults to 52010.
 - **report_server_codes** list (*Optional*): List of codes for the different states.
 
 Note that this basic configuration will only enable you to read the armed/armed away/disarmed status of your alarm and will **not** update the status if the alarm is triggered. This is because of how Egardia built their system. The alarm triggers normally go through their servers.
 You can change this, however, using the following procedure. This is a more advanced configuration.
 
 1. Log in into your alarm system's control panel. You will need to access http://[IP of your control panel]. You know this already since you need it in the basic configuration from above. Log in to the control panel with your Egardia/Woonveilig username and password.
-2. Once logged in, go to *System Settings*, *Report* and change the Server Address for your primary server to the IP or hostname of your Home Assistant machine. Also, update the port number 85 or to anything you like. The provided software that you will set up in the next steps runs on port 85 by default. **Make sure to change the settings of the primary server otherwise the messages will not come through. Note that this will limit (or fully stop) the number of alarm messages you will get through Egardia's / Woonveilig services.** Maybe, that is just what you want. Make sure to save your settings by selecting 'OK'.
+2. Once logged in, go to *System Settings*, *Report* and change the Server Address for your primary server to the IP or hostname of your Home Assistant machine. You can leave the port number set to 52010 or change it to anything you like. **Make sure to change the settings of the primary server otherwise the messages will not come through. Note that this will limit (or fully stop) the number of alarm messages you will get through Egardia's / Woonveilig services.** Maybe, that is just what you want, because it removes the need to pay for their subscription. Make sure to save your settings by selecting 'OK'.
```

Comment:
I'm not sure if we should mention the subscription. Maybe we should leave it out to stay neutral?

Reply:
it's kind of the whole purpose of the component, but sure, I am fine with leaving it out. I expect people to be able to read between the lines.